### DISCUSSION TITLE
what happened to the hoarder-workers image container

### DISCUSSION THREAD

**Discussion by alx-xlx** [karakeep-app/karakeep#695] [Q&A]
i have been running hoarder for few months now, and today i see that the new docker-compose.yml file has a missing **hoarder-workers** and the **redis** container configs. 

I am wondering why that is the case

https://github.com/hoarder-app/hoarder/blob/main/docker/docker-compose.yml

https://github.com/hoarder-app/hoarder/pkgs/container/hoarder-workers

**Comment by MohamedBassem** [comment]
hey, they are no longer needed. Check the upgrade guide in v0.16: https://github.com/hoarder-app/hoarder/releases/tag/v0.16.0

**Comment by MohamedBassem** [comment]
Also note that the name of the hoarder web container changed
